src/default-config-backend.cpp

```diff
@@ -53,7 +53,7 @@ static int handle_config_updated(int fd, uint32_t mask, void *data)
 
     const auto last_slash = config_file.find_last_of('/');
```

Comment:
I guess we can use the built-in std::filesystem functions here to get the base file name? https://en.cppreference.com/w/cpp/filesystem/path/filename